app/models/rubygem.rb

```diff
@@ -7,7 +7,7 @@ class Rubygem < ActiveRecord::Base
   has_many :subscribers, through: :subscriptions, source: :user
   has_many :subscriptions, dependent: :destroy
   has_many :versions, dependent: :destroy, validate: false
-  has_one :latest_version, -> { where(latest: true) }, class_name: "Version"
+  has_one :latest_version, -> { where(latest: true).order(created_at: :desc) }, class_name: "Version"
```

Comment:
Yeah. I think the latest version pushed in the system should be the one showing on the grid/index. That was the behaviour before too, as far as I can tell. 
Good question.